graphrag/llm/openai/openai_chat_llm.py

```diff
@@ -120,6 +120,7 @@ async def _manual_json(
         result = await self._invoke(input, **kwargs)
         history = result.history or []
         output = clean_up_json(result.output or "")
+        output = fix_malformed_json(result.output or "")
```

Comment:
output = fix_malformed_json(output)

Reply:
Your comment seems to have gotten messed up, but I did notice the issue with the output not being forwarded from one function another. Just addressed.